etc/releasing.md

```diff
@@ -264,7 +264,7 @@ Commit the updates to `etc/apidocmenu.md` and `CHANGELOG.md`.
 git commit -m 'Update CHANGELOG for X.Y.Z'
 ```
 
-### Pre-Release Changes PR
+##### Pre-Release Changes PR
```

Comment:
Suggest removing this section header entirely. No reason to separate it from the "... for a Non-Patch Release" section body.